eng/pipelines/templates/jobs/apiview-review-gen-python.yml

```diff
@@ -0,0 +1,59 @@
+pr: none
+
+trigger: none
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+parameters:
+  - name: Reviews
+    type: string
+    default: ''  
+  - name: APIViewURL
+    type: string
+    default: '' 
+  - name: StorageContainerUrl
+    type: string
+    default: ''
+
+variables:
+  NugetSecurityAnalysisWarningLevel: none
```

Comment:
We should include globals.yml from this repo which should include these properties.